### PR TITLE
Moving views and serializers under their own folders

### PR DESCRIPTION
**Pull request by Baranovsky1986**
changed project structure ( mkdir serializers & views )

### CODE REVIEW

File: ratu/serializers/ruo_serializers.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+from rest_framework import serializers
+
+class RuoSerializer(serializers.Serializer):
+    state = serializers.CharField(max_length=100)
```
Comment (criticus) [inline ratu/serializers/ruo_serializers.py:4]: is there any reason why we have max_length only for one field?

File: ratu/views/rfop_views.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+from django.shortcuts import render
+from django.http import HttpResponse
+from rest_framework.response import Response
+from rest_framework.views import APIView
+from ratu.models.rfop_models import Rfop
+from ratu.serializers.rfop_serializers import RfopSerializer
+
+class RfopView(APIView):
+    def get(self, request):
+        rfop = Rfop.objects.all()[:5]
```
Comment (criticus) [inline ratu/views/rfop_views.py:10]: why is there limit 5?

File: ratu/views/ruo_views.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
+from django.shortcuts import render
+from django.http import HttpResponse
+from rest_framework.response import Response
+from rest_framework.views import APIView
+from ratu.models.ruo_models import Ruo
+from ratu.serializers.ruo_serializers import RuoSerializer
+
+class RuoView(APIView):
+    def get(self, request):
+        ruo = Ruo.objects.all()[:5]
```
Comment (criticus) [inline ratu/views/ruo_views.py:10]: same here

File: ratu/urls.py [hunk 4]
```diff
@@ -1,13 +1,11 @@
...
+from ratu.views.ruo_views import RuoView
+from ratu.views.ratu_views import RegionView, DistrictView, CityView, CitydistrictView, StreetView
 
 app_name = 'ratu.apps.RatuConfig'
 
 urlpatterns = [
-    path('', views.index, name='index'),
     path('rfop/', RfopView.as_view()),
     path('ruo/', RuoView.as_view()),
     path('region/', RegionView.as_view()),
```
Comment (Iva-khar) [inline ratu/urls.py:11]: Is it works with new destinations of 'RfopView' & others?

Comment (Baranovsky1986) [reply ratu/urls.py:11]: yes it works

File: ratu/serializers/ruo_serializers.py (hunk 1 above)
Comment (Baranovsky1986) [reply ratu/serializers/ruo_serializers.py:4]: it was in Vital request that you merged before my pr. I changed only (max_length=120) to (max_length=100)

File: ratu/views/rfop_views.py (hunk 2 above)
Comment (Baranovsky1986) [reply ratu/views/rfop_views.py:10]: it was in Vital request that you merged before my pr.
as I understand it, we will change ".objects.all" to filter or anything else, so I think that now it doesn’t matter

File: ratu/views/ruo_views.py (hunk 3 above)
Comment (Baranovsky1986) [reply ratu/views/ruo_views.py:10]: the same^

File: ratu/urls.py (hunk 4 above)
Comment (Iva-khar) [reply ratu/urls.py:11]: ok

**Pull request merged**